packages/nav-frontend-tabell-style/md/Tabell.overview.mdx

```diff
@@ -223,53 +223,70 @@ import "nav-frontend-tabell-style";
 
 ## Sorterbar tabell
 
-Lenker som er barn av `th`-elementene blir stylet til å være helt utfyllende og får 2 små chevrons som peker opp og ned for å indikere at kolonnen kan sorteres i vilkårlig rekkefølge. Bruk klassene `tabell__th--sortert-desc` og `tabell__th--sortert-asc` for at lenkene i stedet skal vise en enkelt chevron i den retningen kolonnen er sortert. Bruk klassen `tabell__td--sortert` på cellene i den sorterte kolonnen for å gi disse en svak grå bakgrunn.
+Lenker som er barn av `th`-elementene blir stylet til å være helt utfyllende og `span` som barn av lenker får to små chevrons som peker opp og ned for å indikere at kolonnen kan sorteres i vilkårlig rekkefølge. Bruk klassene `tabell__th--sortert-desc` og `tabell__th--sortert-asc` for at lenkene skal indikere hvilken retning kolonnen er sortert. Bruk klassen `tabell__td--sortert` på cellene i den sorterte kolonnen for å gi disse en svak grå bakgrunn.
 
 <Alertstripe type="advarsel">
   Husk at du må legge til både <InlineCode>aria-sort</InlineCode> og{" "}
   <InlineCode>role="columnheader"</InlineCode> på alle tabell-headere som skal
   sorteres. Bruk <InlineCode>none</InlineCode>/
   <InlineCode>descending</InlineCode>/<InlineCode>ascending</InlineCode> som
   verdier til <InlineCode>aria-sort</InlineCode> alt ettersom.
+
+
+Husk også å legge til en `<span/>` i lenken for å få rendret chevrons!
```

Comment:
Kan vi løse chevron som et `:after`-element på button istedenfor å kreve at folk legger til en span?

Reply:
Har prøvd å løse det på den måten, men har ikke funnet en løsning for å få plassert begge chevrons 0.5rem etter teksten da. Top-chevron er :before og Bot-chevron er :after så er usikker på om det er i det hele tatt mulig.

Reply:
Det kan godt hende det ikke lar seg gjøre. Jeg tar en titt etter møtet vårt i dag.